applications/clouddemo/netcore/runTests.ps1

```diff
@@ -0,0 +1,22 @@
+﻿# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+& dotnet publish -c Release -f netcoreapp3.1 | Out-Default
+
+if ($LastExitCode -ne 0)
+{
+    exit $LastExitCode
+}
```

Comment:
```suggestion
dotnet build 2>&1 | %{ "$_" }
```
This should be enough. Also, we need redirecting stderr to stdout so that we get all the error messages properly. That's what  the `2>&1 | %{ "$_" }`.